pages/Home.js

```diff
@@ -1,27 +1,78 @@
+'use strict';
+
 import React from 'react';
-import { StyleSheet, Text, View, Button } from 'react-native';
+import { StyleSheet, 
+  Text,
+  View, 
+  Image,
+  ScrollView,
+  ImageBackground,
+  StatusBar,
+  Dimensions } from 'react-native';
+import { Button } from 'react-native-paper';
+import {HomeCSS} from './../Style'
+
+import bgImage from './../assets/home_bg.jpg'
 
+const win = Dimensions.get('window');
+const width = win.width;
+const height = win.height;
+const ratio = win.width/732;
 export default class Home extends React.Component {
   render() {
     return (
-      <View style={styles.container}>
-        We have no friends!
-        <Button
-          title="Add some friends"
-          onPress={() =>
-            this.props.navigation.navigate('Friends')
-          }
-        />
+      <View style = {HomeCSS.container}>
+        <StatusBar backgroundColor="blue" barStyle="light-content" />
+        <ImageBackground source={bgImage} style={HomeCSS.backgroundImage}>
+          <View style ={HomeCSS.homepageContainer}>
+             <Text style ={HomeCSS.heading}>Join with us</Text>
+             <Text style ={HomeCSS.subHeading}>Ready | Cook | Eat</Text>
+             <Button 
+              mode="contained"
+              style = {{marginTop:30}}
+              theme={{
+                roundness: 5,
+                width: 200,
+                colors: {
+                  primary:'#fff',
+                }
+              }}
+              contentStyle = {{width:200,height:60}}
+              onPress={() => alert('Pressed')}>
+              Let's Cook
+            </Button>
+          </View>
+          
+        </ImageBackground>
+
       </View>
+      
     );
   }
 }
 
 const styles = StyleSheet.create({
```

Comment:
Those style can be removed since we are using separate CSS and calling it in Home.js
import {HomeCSS} from './../Style' this pages realated CSS can be using in thsi file itself.